cli/azd/resources/scaffold/templates/resources.bicept

```diff
@@ -288,7 +288,7 @@ module {{bicepName .Name}}CreateConnectionToPostgreSql 'br/public:avm/res/resour
     }
     runOnce: false
     retentionInterval: 'P1D'
-    scriptContent: 'apk update; apk add g++; apk add unixodbc-dev; az extension add --name containerapp; az extension add --name serviceconnector-passwordless --upgrade; az containerapp connection create postgres-flexible --connection appConnectToPostgres --source-id ${ {{bicepName .Name}}.outputs.resourceId} --target-id ${postgreServer.outputs.resourceId}/databases/${postgreSqlDatabaseName} --client-type springBoot --user-identity client-id=${ {{bicepName .Name}}Identity.outputs.clientId} subs-id=${subscription().subscriptionId} user-object-id=${connectionCreatorIdentity.outputs.principalId} -c main --yes;'
+    scriptContent: 'apk update; apk add g++; apk add unixodbc-dev; az extension add --name containerapp; az extension add --name serviceconnector-passwordless --upgrade; az containerapp connection create postgres-flexible --connection {{bicepName .Name}}_Postgres_${resourceToken} --source-id ${ {{bicepName .Name}}.outputs.resourceId} --target-id ${postgreServer.outputs.resourceId}/databases/${postgreSqlDatabaseName} --client-type springBoot --user-identity client-id=${ {{bicepName .Name}}Identity.outputs.clientId} subs-id=${subscription().subscriptionId} user-object-id=${connectionCreatorIdentity.outputs.principalId} -c main --yes;'
```

Comment:
Add `{{bicepName .Name}}` and random suffix to avoid name conflict.

Reply:
In my understanding, the .Name should not have duplicated value. Could you please give me the example that it duplicated?

Reply:
Updated.